common/components/chrome/MainHeader.jsx

```diff
@@ -14,7 +16,7 @@ class MainHeader extends React.PureComponent<{||}> {
         <span onClick={this._onHomeButtonClick}>
           <img
             className="MainHeader-logo"
-            src="https://i.imgur.com/NhoAjjN.png"
+            src={cdn.image("dl_logo.png")}
```

Comment:
Just an aside here but this a 110kb 2040x612 image being displayed at 200x54. Would be worth some filesize and loadtime saving to upload a native 200x54 copy.

Reply:
Thanks for pointing this out.  I've updated the cdn image.